### DISCUSSION TITLE
Suggestion: 'no-data' property for VSelect to make it work without dependency on `items`

### DISCUSSION THREAD

**Discussion by 3dyuval** [vuetifyjs/vuetify#18610] [Ideas]
Hello. 
In my company we use the select field for most of our inputs.
Using this pattern we were able to create a consistent design within a big and modular rule builder with lots and lots of inputs:

```
 <v-select
        :items="[{}]"
        placeholder="Enter date"
        :model-value="formattedDate"
        :menu-props="{class: 'date-picker-menu'}"
        prepend-inner-icon="mdi-calendar"
      >
        <template #item>
          <v-date-picker v-model="date" />
        </template>
      </v-select>

```

Here's a [demo](https://play.vuetifyjs.com/#eNqNU8Fu2zAM/RXOOzgBIqdYBwzwnAwDtgG779YUhWoxsTBZEiTZaRD430fZ8WJvLdpcYpOP5HvP5N058a5cf7U2axtM8qTwpZM2gMfQWFBcHza7JPhdst1pAFlb4wKcweF+BaWpbRNQQAd7Z2pIqUW60xFYGu0DCB4QNhG8WE7je+NqHqjy2wAYGy0WS9j0g2Jl1nLVIHwBjUf4qYPKIvyXrPFHX79IBe55o0K6gnMsij/BTzmkH5iQB0mJMVwbHarnEifkjuK6qdHJcox3RHfIZwPXxZXQEnJoNI2WGkWEEbZYD7YR9yJgbRWhex1Fy7i1/WP/QvoDp0J3CfVBjwrLMAYAckk9PPl+d+7ud8k1QX1LrIwS6Cj7XQd0vVNTTF4bgYr1VAk0s3qOQ90w64yNk86l4t6TEbEds7L8jY5FRNrNCDi0qAWTmiQwSWqothaSlVxRnLsr+K9AkjhaAu+jskmmlz8ZCS3r6VPXQdcMO+sUZFC4hWI98/tZaIWcHHsb9rJSL4KL9YzwVOX/BRE8fNxxAygwX4EYGTZkVk+vPpxIYMRkc48uu36UIq50o+lU4d1wmVyHz0P20TwxX3Fhji9AuuEgqbdp0Sl+yv799LCdZB8eIm/UdPzDgJo/sQrloQqvDpiS34KQbb6XzgdWVlKJsZ+QnsTT8WqjcWxAd3VxIelWyW12m31K4v/H7IY9YuDZTXL/B8Yrgyw=) to show how its all fitting together nicely.

What I'm basically thinking is this, if you add an 'no-data'  property to VSelect to force it to work without any items.
Now it has the ability to show anything you like inside`template #no-data`
Maybe this example or this thinking process will help anyone who was thinking of creating custom drop downs like this. 
If you find it useful, please share your thoughts.
```
<v-select 
no-data
:model-value="displayValue"
>
<template #no-data>
...anything you like
</template>
</v-select>

```

**Comment by KaelWD** [comment]
[Did you try it?](https://play.vuetifyjs.com/#eNqNU8GO0zAQ/ZUhHNJKdSpxLGkREiBx50bRysTTxsKxLdtJt6r674zthibsrnZziJKZ92beG3t+XgrvmvVna6uhx2JT1L5x0gbwGHoLiuvjdl8Evy92ew0gO2tcgAs4PKygMZ3tAwq4wsGZDkoqUe51BDZG+wCCB4RtBC+W0/jBuI4HYn7JgLHQYrGEbWoUmdXAVY/wCTSe4LsOqorwH7LDb4m/KAUeeK9CuYJLJMVH8PMGyg9MyKOkxBjujA7tc4kzckdx3XfoZDPGryQ356usdXEXtIQN9JpaS40iwghbr/PYSHsdsLOK0MlHPTBubfpMP+Q/cCK6WygFPSpswhgAIHqDrVECHQ3/qw7o0kD2xR2z6YxAxZIiAs0mOseh7pl1xnqCXRrFvSe/sRyzsvmDjkVEeZ2SrEOLWjCpSSmTJJq4nZCs4Yri3N3B/3yQk9E5vNeGUQc+SSajk64wsOSACmdrM+ysWJBB4Q7q9Wyyz0Jb5DS0t2Fvl+dFcL2eCZ4afUqI4HyM41lTYH7YMZLvwoxPvz6cyWDEVPMZ3W71SYp4eXtNSwnv8g5yHT7m7G/zyHzLhTm9ALnm1aPaZkCn+Ln6//RhN8k+PETdqGnNc4OOP7IW5bENrzaYit+BkMPmIJ0PrGmlEmM9IT2ZpzXVRuNYgDboNoXiutK9Uun16y83L3vt)
